ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -72,6 +73,7 @@ public void handleTouchEvent(MotionEvent ev, EventDispatcher eventDispatcher) {
       // {@link #findTargetTagForTouch} to find react view ID that will be responsible for handling
       // this gesture
       mChildIsHandlingNativeGesture = false;
+      mGestureStartTime = ev.getEventTime();
```

Comment:
Before, this was getDownTime -- theoretically this should be the same for this event, but let's keep it as getDownTime so we're changing less.